### DISCUSSION TITLE
noDRM - GitHub restriction

### DISCUSSION THREAD

**Discussion by jinmshu1** [noDRM/DeDRM_tools#309] [Q&A] [answered]
Somewhere in the past few months I think I read where GitHub is restricting noDRM?

I've searched discussions but can't find any related info.  If true, what does that mean?

Thanks for any replies!

**Comment by ElleKayEm** [comment, marked as answer]
There was a takedown request from Readium which was resolved.  The repo was offline for a bit while that part of the code was removed.  More here: https://github.com/noDRM/DeDRM_tools/issues/18

**Reply by jinmshu1** [reply]
ElleKayEm,

Thank you!

**Reply by jinmshu1** [reply]
Just another newbie question: does that mean no noDRM plugin updates?

**Reply by ElleKayEm** [reply]
There have been updates since then.

**Reply by JSWolf** [reply]
> There have been updates since then.

But no now compiled version of DeDRM. There are a lot of people who don't know about the updated code or how to update the code from what's there.

**Reply by ElleKayEm** [reply]
I was refering to the release of 10.0.3.  But, yeah, there have been more updates since then too.

**Reply by JSWolf** [reply]
> I was refering to the release of 10.0.3. But, yeah, there have been more updates since then too.

Do you (or anyone else) know how to create a package from the code? I know there;s a way to do it  on GitHub. I just don't know how.

**Reply by jinmshu1** [reply]
Again, newbie. Is there no way to PM on GitHub? Nothing much to say, but just couldn't find a way.

**Reply by ElleKayEm** [reply]
@JSWolf  Click Actions near the top of the page here.  Click the top (most recent) workflow run on that page.  Scroll to the bottom of that page and click "plugin" to download.  You'll need to unzip that a couple times.  And then you'll see the 2 plugins and 3 readme files.

**Reply by jinmshu1** [reply]
"here" isn't linked.

**Reply by ElleKayEm** [reply]
@jinmshu1 No, I don't think there is.

**Reply by ElleKayEm** [reply]
> "here" isn't linked.

Here on this page.  Or any page in this repo.

**Reply by JSWolf** [reply]
> @JSWolf Click Actions near the top of the page here. Click the top (most recent) workflow run on that page. Scroll to the bottom of that page and click "plugin" to download. You'll need to unzip that a couple times. And then you'll see the 2 plugins and 3 readme files.

Thank you. That's perfect!

**Reply by jinmshu1** [reply]
I finally understand! Thank you!
